### DISCUSSION TITLE
List all dependencies

### DISCUSSION THREAD

**Discussion by nico-robert** [apnadkarni/tcl-cffi#114] [General]
Hello @apnadkarni ,
I would like to know if it is possible if `tcl-cffi` is able to list all dependencies in my shared library . For example in `libharu` (always...) I know that if `libpng` is not linked some functions will not work.
I know this command line on MacOS `otool` `-L` mylib.dylib (not at all on Linux/Windows...).
It would be possible ?

Thanks

Nicolas

**Comment by apnadkarni** [comment]
Unfortunately not directly. Also as far as I know, it would also be a fair amount of work to do for multiple platforms.

/Ashok

**Reply by nico-robert** [reply]
ok , thank you
